hack/deploy_traffic_flow_tests.sh

```diff
@@ -1,3 +1,5 @@
+#!/bin/bash
```

Comment:
```
/usr/bin/env bash
```

Reply:
I choose `#!/bin/bash` intentionally and I am away of `env` (and decided intentionally against it for various reasons).

If you suggest an alternative, then it would be useful to leave a hint for why that is preferable.

OK, changed.